sdk/identity/azure-identity/tests/test_cli_credential.py

```diff
@@ -0,0 +1,47 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+import json
+
+from azure.identity import CliCredential
+from azure.core.exceptions import ClientAuthenticationError
+
+try:
+    from unittest.mock import Mock
+except ImportError:
+    from mock import Mock
+
+
+def test_cli_credential():
+    access_token = '***'
+    expires_on = '9999-1-1 00:00:00.1' 
+
+    cred = CliCredential()
+    cred._get_proc_stdout = Mock(return_value=json.dumps({"accessToken" : access_token, "expiresOn" : expires_on}))
```

Comment:
I suggest using [mock.patch](https://docs.python.org/3.5/library/unittest.mock.html#patch) to mock `check_output` instead. Then you could have more realistic tests that don't modify the code they test.

Reply:
Code pushed.